pallet-logion-loc/src/mock.rs

```diff
@@ -169,6 +169,18 @@ parameter_types! {
         reserve_percent: Percent::from_percent(0),
         treasury_percent: Percent::from_percent(100),
     };
+    pub const CollectionItemFeeDistributionKey: DistributionKey = DistributionKey {
+        stakers_percent: Percent::from_percent(25),
+        collators_percent: Percent::from_percent(25),
+        reserve_percent: Percent::from_percent(25),
+        treasury_percent: Percent::from_percent(25),
```

Comment:
I just realize I forgot to update the story. Fees are distributed as follows:

- 3% go to the LOC's owner
- 9% are distributed among the selected VIs
- The remainder go to Logion treasury

The payment of VIs is not yet something covered so we may keep this for later. Regarding the payment of the owner and other beneficiaries, I think we already implemented a mechanism handling this, so it would be nice to already implement it (just send 3% to LLO and 97% to the treasury to begin with).

Reply:
Done as specified.